CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 1.5.0
 Release Date: Not published yet
 
+- Fixed `StateKeyValueStoreController` method name from `items` to `keys`. It has no effect in API route (preserved as `/state/key-value-store/keys/`) but prometheus label `action` for `http_requests_received_total` metric will change from `items` to `keys` for this endpoint.
```

Comment:
I wouldn't state it as "fixed" as nothing was fundamentally broken after all. This is a change, not a bugfix. 

Just thinking out loud: do we want to review controller/action names in general?